### ISSUE TITLE
Wordpress problem with new filter - wrong URL construction 

### ISSUE THREAD

**Issue by gngn23** [adixon/ca.civicrm.logviewer#33]
Unfortunately the new filtering from my PR #31 does not work with Wordpress.

The URL in templates/CRM/Logviewer/Page/LogViewer.tpl is hardwired to 'civicrm/admin/logviewer':
`<form action="/civicrm/admin/logviewer" ...>`

It should use the correct URL construction via CRM_Utils_System::url (in this case the variant for Smarty):
`<form action="{crmURL p='civicrm/admin/logviewer'}" ...>`

Sorry about that, patch coming.

**Comment by gngn23** [comment]
@adixon : The form URL  `/civicrm/admin/logviewer` (which is correct for Drupal systems) results in _access denied_ on Wordpress. Because of this filtering *does not work at all on WP* (but gets you an access denied error).

I'm angry at myself for not testing with WP :( 

With the small change in the commit and in PR #34 the URL is constructed correctly for both CMS. 

So it would be great if the commit would soon make it into a new version.

**Comment by adixon** [comment]
New bugfix version released! Who wants angry WP users?

**Comment by gngn23** [comment]
@adixon when I download "Source code (zip)"  from https://github.com/adixon/ca.civicrm.logviewer/releases/tag/2.2.1 the info.xml still contains `<version>2.2.0</version>`.
Same in https://github.com/adixon/ca.civicrm.logviewer/blob/master/info.xml
I think you forgot to change info.xml - or am I getting something wrong?

**Comment by adixon** [comment]
Nope, you are correct. I generated that bugfix release via the web interface because I was in a hurry. I'm still learning!

**Comment by adixon** [comment]
I hope I did it right this time. Still managed to do it all via the web ui.
